File: informe 1/5.py
def calcular_cargo(horas):
    if horas <= 1:
        cargo = 3.0
    else:
        cargo = 3.0 + (horas - 1) * 0.5
    return min(cargo, 12.0)

h = int(input("Ingrese el número de horas: "))
print("El cargo es: S/", calcular_cargo(h))

File: informe 1/5.cpp
#include <iostream>
using namespace std;
float calcularCargo(int horas) {
    float cargo;
    if (horas <= 1) {
        cargo = 3.0;
    } else {
        cargo = 3.0 + (horas - 1) * 0.5;
    }
    if (cargo > 12.0) {
        cargo = 12.0;
    }
    
    return cargo;
}
int main(){
    int horas;
    cout << "Ingrese el número de horas de estacionamiento: ";
    cin >> horas;
    cout << "El cargo es: S/" << calcularCargo(horas) << endl;
    return 0;
} 